docs/assets/scripts/docs.js

```diff
@@ -1,4 +1,38 @@
 $(function() {
+  function filterOrderSideNavItem(pageUrl, items) {
+    var ORDER = [
+      'ShopifyBuy',
+      'ShopClient',
+      'ProductModel',
+      'ProductVariantModel',
+      'ProductOptionModel',
+      'CartModel',
+      'CartLineItemModel'
+    ];
+
+    var newItems = ORDER.map(function(key) {
+      return items.find(function(docClass) {
+        return docClass.name === key;
+      });
+    })
```

Comment:
Hey assuming in this case it's not adding a space after `function` anything else. Linting is totally broken on this file since it's es5.